src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java

```diff
@@ -45,80 +44,72 @@ protected String getPackageLocation() {
-            checkConfig,
+        verifyWithInlineConfigParser(
             getPath("InputJavadocPackageBadCls2.java"),
             expected);
     }
 
     @Test
     public void testWithMultipleFiles() throws Exception {
-        final Configuration checkConfig = createModuleConfig(JavadocPackageCheck.class);
-        final String path1 = getPath("InputJavadocPackageNoJavadoc.java");
-        final String path2 = getPath("InputJavadocPackageBadTag.java");
         final String[] expected = {
-            "1: " + getCheckMessage(MSG_PACKAGE_INFO),
+            "9: " + getCheckMessage(MSG_PACKAGE_INFO),
         };
-        verify(
-            createChecker(checkConfig),
-            new File[] {new File(path1), new File(path2)},
-            path1,
-            expected);
+        verifyWithInlineConfigParser(
+                getPath("InputJavadocPackageNoJavadoc.java"),
+                expected);
+        verifyWithInlineConfigParser(
+                getPath("InputJavadocPackageBadTag.java"),
+                expected);
     }
 
     @Test
     public void testBoth() throws Exception {
-        final Configuration checkConfig = createModuleConfig(JavadocPackageCheck.class);
         final String[] expected = {
-            "1: " + getCheckMessage(MSG_LEGACY_PACKAGE_HTML),
+            "9: " + getCheckMessage(MSG_LEGACY_PACKAGE_HTML),
         };
-        verify(checkConfig,
+        verifyWithInlineConfigParser(
             getPath("bothfiles" + File.separator + "InputJavadocPackageBothIgnored.java"),
             expected);
     }
 
     @Test
     public void testHtmlDisallowed() throws Exception {
-        final Configuration checkConfig = createModuleConfig(JavadocPackageCheck.class);
         final String[] expected = {
-            "1: " + getCheckMessage(MSG_PACKAGE_INFO),
+            "9: " + getCheckMessage(MSG_PACKAGE_INFO),
         };
-        verify(checkConfig,
-            getPath("pkghtml" + File.separator + "InputJavadocPackageHtmlIgnored.java"), expected);
+        verifyWithInlineConfigParser(
+            getPath("pkghtml" + File.separator + "InputJavadocPackageHtmlIgnored.java"),
+                expected);
     }
 
     @Test
     public void testHtmlAllowed() throws Exception {
-        final DefaultConfiguration checkConfig = createModuleConfig(JavadocPackageCheck.class);
-        checkConfig.addProperty("allowLegacy", "true");
         final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
-        verify(createChecker(checkConfig),
-            getPath("pkghtml" + File.separator + "InputJavadocPackageHtmlIgnored2.java"),
-            getPath("pkghtml" + File.separator + "package-info.java"), expected);
+        verifyWithInlineConfigParser(
+                getPath("pkghtml" + File.separator + "InputJavadocPackageHtmlIgnored2.java"),
+                expected);
+        verifyWithInlineConfigParser(
+                getPath("noparentfile" + File.separator + "package-info.java"),
+                expected);
```

Comment:
no, it is not same.
please create new method verifyWithInlineConfigParser that takes few paths in same time as it is in `verify` method.